packages/test/service-load-test/src/utils.ts

```diff
@@ -53,7 +54,8 @@ class FileLogger implements ITelemetryBufferedLogger {
         this.baseLogger?.send(event);
 
         event.Event_Time = Date.now();
-        Object.keys(event).forEach((k)=>this.schema.add(k));
+        // keep track of the frequency of every log event, as we'll sort by most common on write
```

Comment:
> we'll sort by most common on write

?? Why wouldn't we just log in chronological order...?

Reply:
this is just the schema, not the entries. the entries are chronological. ordering the schema by frequency makes the logs easier to read

Reply:
Ah ok, I glossed over this code and hadn't grok'd what schema was. That makes sense.